packages/fuel-indexer-api-server/src/api.rs

```diff
@@ -113,6 +114,7 @@ impl GraphQlApi {
         let sm = SchemaManager::new(pool.clone());
         let schema_manager = Arc::new(RwLock::new(sm));
         let config = config.clone();
+        let max_body_limit = config.max_body_limit;
```

Comment:
Nit: I do think the naming is interesting. It effectively has 2 nouns that imply the same thing "max" and "limit". Not a blocker, can always update later.

Reply:
true, I agree. Let me update all occurences to `max_body`